src/Microsoft.AspNetCore.Razor.Language/Intermediate/TagHelperIRNode.cs

```diff
@@ -46,6 +47,8 @@ public override RazorDiagnosticCollection Diagnostics
 
         public TagMode TagMode { get; set; }
 
+        public ICollection<TagHelperDescriptor> TagHelpers { get; set; }
```

Comment:
Thinking it's probably better to treat this guy like Children, Diagnostics, Annotations etc. and have a getter only.